src/platform/user/profile/vet360/util/local-vet360.js

```diff
@@ -227,6 +227,35 @@ export default {
       1000,
     );
   },
+  addressValidationSuccessSingleLowConfidenceSuggestion() {
+    return asyncReturn(
+      {
+        addresses: [
+          {
+            address: {
+              addressLine1: '400 N 65th St',
+              addressType: 'DOMESTIC',
```

Comment:
Just curious, what is the purpose of this data? My initial impression is that it should live in /test..but then, I'm unfamiliar with this work.